test/TEST-78-SIGQUEUE/Makefile

```diff
@@ -0,0 +1 @@
+../TEST-01-BASIC/Makefile
```

Comment:
new tests are sort of expensive, maybe it could be merged with `TEST-20-MAINPIDGAMES`?

Reply:
this shouldn't be expensive. some time ago zbigniew reworked things so that most integration test scan be run within the same image, except for a bunch that are excluded. this one should be a regular one.

Reply:
I did that :-) But there is still a cost to boot the image. We are often timing out on jammy-amd64 as it takes more than 4hr to run. I'd really like to coalesce a bit the integration tests, and have them 'per component' or so, rather than per test case.